docs/src/snap/howto/cis-hardening.md

```diff
@@ -0,0 +1,270 @@
+# CIS compliance 
+
+CIS Hardening refers to the process of implementing security configurations that
+align with the benchmarks set by the [Center for Internet Security (CIS)][]. The
+open source tool [Kube-bench][] is designed to automatically check whether
+your Kubernetes clusters are configured according to the 
+[CIS Kubernetes Benchmark][]. This guide covers how to:
+
+- [Install and configure kube-bench](#install-kube-bench)
```

Comment:
An in-page index is rendered on the right of each page. These sort of 'top-of-page index' constructs aren't very useful in docs, they just make the pages longer. Appreciate you may need to rework the intro

Reply:
Index removed and intro reworked